alacritty/src/event.rs

```diff
@@ -1207,7 +1207,8 @@ impl input::Processor<EventProxy, ActionContext<'_, Notifier, EventProxy>> {
                     self.ctx.display.cursor_hidden = false;
                     *self.ctx.dirty = true;
                 },
-                EventType::Message(message) => {
+                // Add message only in case it's not already queued.
```

Comment:
```suggestion
                // Add message only if it's not already queued.
```

Doesn't really matter, but this is a little shorter and reads nicer imo.